package.json

```diff
@@ -86,7 +86,7 @@
     "broccoli-string-replace": "^0.1.1",
     "chalk": "^1.1.0",
     "electron-forge": "^2.8.0",
-    "electron-protocol-serve": "^1.1.0",
+    "electron-protocol-serve": "^1.2.0-2",
```

Comment:
Er, I mean, I don't know what you're talking about, there are no backticks there (yay rebasing 😉 )